config/routes.rb

```diff
@@ -13,8 +13,11 @@
     end
   end
 
+
   namespace :audits do
-    get '/', to: "audits#index"
+    get '/', to: "allocations#index"
+    get '/my_content', to: "audits#index", as: 'my_content'
```

Comment:
The more "URL-like" thing to do is use spinal-case rather than snake_case.

Reply:
Thanks 👍